share_extension/android/extension_post/extension_post.js

```diff
@@ -3,7 +3,7 @@
 
 import React, {PureComponent} from 'react';
 import {NavigationActions} from 'react-navigation';
-import TouchableItem from 'react-navigation/src/views/TouchableItem';
+import TouchableItem from 'react-navigation-stack/dist/views/TouchableItem';
```

Comment:
Do we need to add `react-navigation-stack` to package.json?
Should we delete other instance of `react-navigation/src/views/TouchableItem` at `package/moduleNames` and `package/modulePaths`?

Reply:
@saturninoabril we should not add that as is part of react-navigation

And for the moduleNames/modulePaths there is another ticket for that